charts/flink-job/values.yaml

```
@@ -312,6 +312,8 @@ metrics:
     targetLabels:
       - component
       - cluster
+    # -- JobLabel selects the label from the associated Kubernetes service which will be used as the job label for all metrics
```

Just to clarify that it is a Kubernetes resource which is referenced
```suggestion
    # -- JobLabel selects the label from the associated Kubernetes Service resource which will be used as the job label for all metrics
```